index.js

```diff
@@ -217,7 +217,7 @@ module.exports = {
     "arrow-spacing": 0,
     "constructor-super": 2,
     "generator-star-spacing": 0,
-    "no-arrow-condition": 0,
+    "no-confusing-arrow": 2,
```

Comment:
Rule was deprecated but this makes the following code an error:

```
var x = a => 1 ? 2 : 3;
var x = (a) => 1 ? 2 : 3;
var x = (a) => (1 ? 2 : 3);
```

see https://eslint.org/docs/rules/no-confusing-arrow for more info